### PR TITLE
feat: add personnel

### PR DESCRIPTION
**Pull request by 1emu**
image

image

image

image

### CODE REVIEW

File: src/components/Projects/ActionablePersonnelView.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  const [isFormDisabled, setIsFormDisabled] = useState(false)
+
+  const handleAddPersonnel = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault()
+    setShowCreatePersonnelForm(true)
+  }
+
+  const handleSavePersonnel = async (personnel: PersonnelAttributes) => {
+    setIsFormDisabled(true)
+    const newPersonnel = await Governance.get().createPersonnel({ ...personnel, project_id: projectId }) //TODO: try/catch handle error vs success
```
Comment (ncomerci) [inline src/components/Projects/ActionablePersonnelView.tsx:41]: 👀👀

File: src/components/Projects/ActionablePersonnelView.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    console.log('newPersonnel', newPersonnel)
```
Comment (ncomerci) [inline src/components/Projects/ActionablePersonnelView.tsx:42]: log here

**Review by ncomerci — APPROVED** [review]

File: src/components/Projects/ActionablePersonnelView.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                <></>
+              )
+            }
+          />
+        ),
+      })),
+    [members, isEditor, t]
+  )
+
+  const addressCheck = (data: string) => !data || data.length === 0 || (!!data && isEthereumAddress(data))
```
Comment (andyesp) [inline src/components/Projects/ActionablePersonnelView.tsx:84]: This function can be outside component.

File: src/components/Projects/ActionablePersonnelView.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  const personnelSchema: ZodSchema<Pick<PersonnelAttributes, 'name' | 'address' | 'role' | 'about' | 'relevantLink'>> =
```
Comment (andyesp) [inline src/components/Projects/ActionablePersonnelView.tsx:85]: This one too

File: src/components/Projects/ActionablePersonnelView.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    z.object({
+      name: z.string().min(1, 'Name is required').max(80),
+      address: z.string().refine(addressCheck, { message: 'Invalid address' }),
+      role: z.string().min(1, 'Role is required').max(80),
+      about: z.string().min(1, 'About is required').max(750),
+      relevantLink: z.string().min(0).max(200).url().optional().or(z.literal('')),
+    })
+
+  const personnelFields: ProjectSidebarFormFields<PersonnelAttributes> = [
```
Comment (andyesp) [inline src/components/Projects/ActionablePersonnelView.tsx:94]: And this array too

**Pull request merged**